e2e/steps_file.js

```diff
@@ -54,7 +54,7 @@ const confirmDetailsPage = require('./fragments/confirmDetails.page');
 
 const applicationTypePage = require('./pages/generalApplication/applicationType.page');
 const hearingDatePage = require('./pages/generalApplication/hearingDate.page');
-const n245FormPage = require('./pages/generalApplication/n245Form.page');
+//const n245FormPage = require('./pages/generalApplication/n245Form.page');
```

Comment:
is this intentional? if so, May I know why we are commenting it out?

Reply:
this is requirement, applicant don't need to upload n245